index.bs

```diff
@@ -3267,7 +3268,7 @@ Maturity Stages on the Recommendation Track</h4>
 			that the document is considered complete and fit for purpose,
 			and that no further refinement to the text is expected
 			without additional implementation experience and testing;
-			additional features in a later revision may however be expected.
+			additional features in a later revision could however be expected.
```

Comment:
'might'? it's not an expression of capability, but of possibility, isn't it?

Reply:
```suggestion
			however, additional features might be expected in a later revision.
```